cmd/whereabouts.go

```diff
@@ -39,13 +46,17 @@ func main() {
 				logging.Errorf("IPAM configuration load failed: %s", err)
 				return err
 			}
-			logging.Debugf("DEL - IPAM configuration successfully read: %+v", *ipamConf)
 
+			startTime = time.Now()
+			logging.Debugf("DEL - IPAM plugin start. Config: %+v", *ipamConf)
 			ipam, err := kubernetes.NewKubernetesIPAM(args.ContainerID, *ipamConf)
 			if err != nil {
 				return logging.Errorf("IPAM client initialization error: %v", err)
 			}
-			defer func() { safeCloseKubernetesBackendConnection(ipam) }()
+			defer func() {
+				safeCloseKubernetesBackendConnection(ipam)
+				logging.Debugf("DEL - IPAM plugin finished. Took: %q Config: %+v", time.Since(startTime).String(), *ipamConf)
```

Comment:
maybe in case of a failure better to change to `finished (failed)`
so we can filter the failure lines easily when benchmarking ?
unless it is not important because when benchmarking we dont run failures cases

if it is - then also on ADD please

Reply:
Yes, there's no point bench marking failed results.

BUT it makes sense to log the result (or a simplified version of it). 

I'll act on this.